content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,315 @@
+=====================================
+
+Even if a quotation is not created directly from the :guilabel:`Alternatives` tab of another |RfQ|,
+it can still be linked to existing |RfQs|.
 
-Then, add information to the :abbr:`RFQ (Request for Quotation)` form: add a vendor from the
-drop-down next to the :guilabel:`Vendor` field, and click :guilabel:`Add a product` to select a
-product from the drop-down menu in the :guilabel:`Product` column. Then, set the desired purchase
-quantity in the :guilabel:`Quantity` column, and change the purchase price in the
-:guilabel:`Unit Price` column, if desired.
+To do that, begin by creating a new |RfQ|. Navigate to :menuselection:`Purchase app --> New`. Fill
+out the |RfQ|, according to the :ref:`previous instructions <purchase/manage_deals/create-rfq>`.
 
-Once ready, click :guilabel:`Send by Email`. This causes a :guilabel:`Compose Email` pop-up window
-to appear, wherein the message to the vendor can be customized. Once ready, click :guilabel:`Send`
-to send an email to the vendor listed on the purchase order form.
+Then, once ready, click the :guilabel:`Alternatives` tab. Since this new |RfQ| was created
+separately, there are no other orders linked yet.
 
-Then, click the :guilabel:`Alternatives` tab once more. Since this new :abbr:`PO (Purchase Order)`
-was created separately, there are no other orders linked yet. To link this order with the
-alternatives created previously, click :guilabel:`Link to Existing RfQ` on the first line in the
-:guilabel:`Vendor` column.
+However, to link this |RfQ| with existing alternatives, click :guilabel:`Link to Existing RfQ` on
+the first line in the :guilabel:`Vendor` column.
 
-.. image:: calls_for_tenders/calls-for-tenders-link-existing-rfq.png
+.. image:: calls_for_tenders/calls-for-tenders-link-rfq-popup.png
    :align: center
-   :alt: pop-up to link new quotation to existing RFQs.
+   :alt: Pop-up to link new quotation to existing RFQs.
 
-This causes an :guilabel:`Add: Alternative POs` pop-up window to appear. Select the three purchase
-orders created previously, and click :guilabel:`Select`. All of these orders are now copied to this
-:abbr:`PO (Purchase Order)` under the :guilabel:`Alternatives` tab.
+This opens an :guilabel:`Add: Alternative POs` pop-up window. Select the desired previously-created
+|RfQs|, and click :guilabel:`Select`. All of these orders are now copied to this |RfQ|, and can be
+found under the :guilabel:`Alternatives` tab.
 
 .. tip::
-   If a large number of purchase orders are being processed and the previous
-   :abbr:`POs (Purchase Orders)` can't be located, try clicking :menuselection:`Group By -->
-   Vendor` under the search bar at the top of the pop-up window to group by the vendors selected on
-   the previous orders.
+   If a large number of |POs| are being processed, and the previous |POs| can't be located, click
+   the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar, at the top of the pop-up
```

Comment:
Remove non breaking space character and replace with a normal space :)
```suggestion
   the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar, at the top of the pop-up
```

Reply:
Suggestion implemented ✅